diff --git a/modules/annlib_adaptbx/SConscript b/modules/annlib_adaptbx/SConscript
index c85217f..b656c09 100644
--- a/modules/annlib_adaptbx/SConscript
+++ b/modules/annlib_adaptbx/SConscript
@@ -41,7 +41,7 @@ if not libtbx.env.module_is_installed("annlib_adaptbx"):
 
   if (env_etc.static_libraries): builder = env.StaticLibrary
   else:                          builder = env.SharedLibrary
-  builder(target='#lib/ann',
+  builder(target='#lib/ann_cctbx',
     source = ["../annlib/src/ANN.cpp",
               "../annlib/src/bd_fix_rad_search.cpp",
               "../annlib/src/bd_pr_search.cpp",
diff --git a/modules/annlib_adaptbx/ann/boost_python/SConscript b/modules/annlib_adaptbx/ann/boost_python/SConscript
index cb4ff77..3e23a01 100644
--- a/modules/annlib_adaptbx/ann/boost_python/SConscript
+++ b/modules/annlib_adaptbx/ann/boost_python/SConscript
@@ -1,6 +1,6 @@
 Import("env_annlib_boost_python_ext")
 env = env_annlib_boost_python_ext.Clone()
-env.Prepend(LIBS=["ann",])
+env.Prepend(LIBS=["ann_cctbx",])
 env.SharedLibrary(
   target="#lib/annlib_ext",
   source=["annlib_ext.cpp",])
diff --git a/modules/cctbx_project/rstbx/SConscript b/modules/cctbx_project/rstbx/SConscript
index 013e6d469d..9be93090e0 100644
--- a/modules/cctbx_project/rstbx/SConscript
+++ b/modules/cctbx_project/rstbx/SConscript
@@ -43,7 +43,7 @@ if not libtbx.env.module_is_installed("rstbx"):
     Import("env_boost_python_ext")
     env_rstbx_boost_python_ext = env_boost_python_ext.Clone()
     env_rstbx_boost_python_ext.Prepend(
-                  LIBS=["rstbx", "ann", "cctbx", "scitbx_boost_python"])
+                  LIBS=["rstbx", "ann_cctbx", "cctbx", "scitbx_boost_python"])
     env_rstbx_boost_python_ext.SharedLibrary(
                   target="#lib/rstbx_ext", source="ext.cpp")
     env_rstbx_boost_python_ext.SharedLibrary(
diff --git a/modules/cctbx_project/rstbx/apps/SConscript b/modules/cctbx_project/rstbx/apps/SConscript
index 21f8e6133d..53665e350b 100644
--- a/modules/cctbx_project/rstbx/apps/SConscript
+++ b/modules/cctbx_project/rstbx/apps/SConscript
@@ -2,7 +2,7 @@ Import("env_rstbx_boost_python_ext", "env_etc")
 
 env = env_rstbx_boost_python_ext.Clone()
 env_etc.enable_more_warnings(env=env)
-env.Prepend(LIBS=["ann","spotfinder"])
+env.Prepend(LIBS=["ann_cctbx","spotfinder"])
 env.SharedLibrary(
   target="#lib/rstbx_integration_ext",
   source=["integration_ext.cpp"])
diff --git a/rstbx/bandpass/SConscript b/rstbx/bandpass/SConscript
index 45ebfc2bc7..71ec1e2747 100644
--- a/modules/cctbx_project/rstbx/bandpass/SConscript
+++ b/modules/cctbx_project/rstbx/bandpass/SConscript
@@ -2,7 +2,7 @@ Import("env_rstbx_boost_python_ext", "env_etc")
 
 env = env_rstbx_boost_python_ext.Clone()
 env_etc.enable_more_warnings(env=env)
-env.Prepend(LIBS=["ann"])
+env.Prepend(LIBS=["ann_cctbx"])
 env.SharedLibrary(
   target="#lib/rstbx_bandpass_ext",
   source=["ext.cpp"])
diff --git a/modules/cctbx_project/spotfinder/SConscript b/modules/cctbx_project/spotfinder/SConscript
index 112c7a440f..7095b797cd 100644
--- a/modules/cctbx_project/spotfinder/SConscript
+++ b/modules/cctbx_project/spotfinder/SConscript
@@ -33,7 +33,7 @@ env_etc.include_registry.append(
   paths=env_etc.spotfinder_common_includes)
 
 envlm = env.Clone(
-  LIBS=["ann","omptbx"]+env_etc.libm
+  LIBS=["ann_cctbx","omptbx"]+env_etc.libm
 )
 
 boost_libs = "#"+os.path.join(os.path.basename(env_etc.boost_dist), "libs")
