diff --git a/modules/annlib_adaptbx/SConscript b/modules/annlib_adaptbx/SConscript
index c85217f..b656c09 100644
--- a/modules/annlib_adaptbx/SConscript
+++ b/modules/annlib_adaptbx/SConscript
@@ -41,7 +41,7 @@ if not libtbx.env.module_is_installed("annlib_adaptbx"):
 
   if (env_etc.static_libraries): builder = env.StaticLibrary
   else:                          builder = env.SharedLibrary
-  builder(target='#lib/ann',
+  builder(target='#lib/ann_cctbx',
     source = ["../annlib/src/ANN.cpp",
               "../annlib/src/bd_fix_rad_search.cpp",
               "../annlib/src/bd_pr_search.cpp",
diff --git a/modules/annlib_adaptbx/ann/boost_python/SConscript b/modules/annlib_adaptbx/ann/boost_python/SConscript
index cb4ff77..3e23a01 100644
--- a/modules/annlib_adaptbx/ann/boost_python/SConscript
+++ b/modules/annlib_adaptbx/ann/boost_python/SConscript
@@ -1,6 +1,6 @@
 Import("env_annlib_boost_python_ext")
 env = env_annlib_boost_python_ext.Clone()
-env.Prepend(LIBS=["ann",])
+env.Prepend(LIBS=["ann_cctbx",])
 env.SharedLibrary(
   target="#lib/annlib_ext",
   source=["annlib_ext.cpp",])
